### PR TITLE
Release 3.8 stable Beta1

### PR DESCRIPTION
**Pull request by eloycoto**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [3.8.0-alpha2] - 2019-02-18
+## [3.8.0-beta1] - 2019-03-07
```
Comment (davidor) [inline CHANGELOG.md]: Shouldn't it be `-rc1` instead?
Also, `version.lua` should be updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,9 +4,12 @@ All notable changes to this project will be documented in this file.
...
 
-### Fixed:
+- Fixed naming issues in policies [THREESCALE-4150](https://issues.jboss.org/browse/THREESCALE-4150) [PR #1167](https://github.com/3scale/APIcast/pull/1167)
+- Fixed issues on invalid config in logging policy [THREESCALE-4605](https://issues.jboss.org/browse/THREESCALE-4605) [PR #1168](https://github.com/3scale/APIcast/pull/1168)
+
+## [3.7.0-alpha2] - 2019-02-18
```
Comment (davidor) [inline CHANGELOG.md]: This looks out of place.

File: gateway/src/apicast/version.lua [hunk 3]
```diff
@@ -1 +1 @@
-return "3.8.0-alpha2"
+return "3.8.0-rc1"
```
Comment (davidor) [inline gateway/src/apicast/version.lua]: rc1 releases can become GA, that's why in the past we've omitted this part.

Comment (thomasmaas) [reply gateway/src/apicast/version.lua]: Would be good if we work towards consequent naming between various components. (i.e. https://github.com/3scale/porta/releases vs https://github.com/3scale/APIcast/releases)

Comment (davidor) [reply gateway/src/apicast/version.lua]: @thomasmaas You mean not having upstream version numbers and only product ones?

Comment (thomasmaas) [reply gateway/src/apicast/version.lua]: No, I mean using "alpa2", lowercase, etc

File: CHANGELOG.md (hunk 2 above)
Comment (davidor) [reply CHANGELOG.md]: This section should appear below the 3.8.x ones right?

**Pull request merged**